transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Documents/API.java

```diff
@@ -2,12 +2,25 @@
 
 import org.springframework.data.mongodb.core.mapping.Document;
 
+@Document
+public class API {
 
-public interface API {
-    String getApiKey();
+    private final String apiKey;
 
-    String getApiVersion();
+    private final String apiVersion;
```

Comment:
Technically not an API version as it represents that chatbot version - Version ID may be more appropriate.